### ISSUE TITLE
Time required for fine tuning on English language dataset

### ISSUE THREAD

**Issue by adfost** [TensorSpeech/TensorflowTTS#139]
I am trying to fine tune FastSpeech/MelGAN on a different English language dataset, but after several hours of training on both, I get sound files showing no progress. Do I just need to train longer? Or is there a deeper error?

**Comment by dathudeptrai** [comment]
@adfost can you share a tensorboard for both?

**Comment by adfost** [comment]
@dathudeptrai I didn't run this on colab (I trained the model on an AWS instance). I do think that the loss fell from about 1 to staying constant at 0.3 for both models.

**Comment by dathudeptrai** [comment]
@adfost hi, tensorboard should be accessible in any case. or can you share me a `event.out.tfevents...` file?

**Comment by adfost** [comment]
I'm going to email you the files, since they are of a supported type.

**Comment by adfost** [comment]
I sent you the files under the previous email conversation. Hopefully they are not misclassified as a virus.

**Comment by adfost** [comment]
I think we can close this issue I pinpointed the problem.
